src/components/interaction-designer/block-editors/ContactPropertySelector.vue

```diff
@@ -71,3 +76,9 @@ class ContactPropertySelector extends mixins(Lang) {
 
 export default ContactPropertySelector;
 </script>
+
+<style lang="css" scoped>
```

Comment:
Let's confirm what @bzabos thinks about it.

Vue-multiselect doesn't use the same approach with Bootstrap for validation. It is using more complicated library https://vuelidate.js.org/

For now, I think this simple css overriding is OK.

Reply:
This was the simplest solution I could think of instead of installing a new library and configuring it. Also, it does not give the same styling(red box) as bootstrap.